### PR TITLE
Fix Docker build - Create home directories for accounts

### PR DESCRIPTION
**Pull request by tfmorris**

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -35,10 +35,12 @@ docker-machine start default # Start the docker daemon
...
 
+Building the images below requires about 6 GB of space for Docker.
+
 # build images
-docker build -t olbase:latest -f docker/Dockerfile.olbase . # 30+ min (Win10Home/Dec 2018)
-docker-compose build web # 10+ min (Win10Home/Dec 2018)
-docker-compose build solr # 5+ min (Win10Home/Dec 2018)
+docker build -t olbase:latest -f docker/Dockerfile.olbase . # ~4 minutes on a 3 yr old Macbook Pro (Dec 2019)
+docker-compose build web # 1min15sec
+docker-compose build solr # 1 minute
```
Comment (cdrini) [inline docker/README.md]: These timings look really short; docker caches things pretty aggressively, so unless you uninstalled everything and started from scratch, my guess would be your computer had cached some of the images (so these timing aren't taking into account e.g. download time).

In general, I think these timings should be quite liberal; I don't want someone to expect something to take 4 minutes and have it take 30+; that sucks :P However expecting something to take 30+ minutes and having it take 4 is much better. I'd recommend _adding_ your timings, like so:

```
docker build -t olbase:latest -f docker/Dockerfile.olbase . # 30+ min (Win10Home/Dec 2018); 5 min (MacBook Pro/Dec 2019)
```

But _only_ if you've done a _full_ reset; these timings are mainly for first-comers.

**Review by cdrini — CHANGES_REQUESTED** [review]
Can you expand on what error you observed? I'm not sure why these users need home directories.

File: docker/README.md (hunk 1 above)
Comment (cdrini) [reply docker/README.md]: (Also, although a MacBook Pro is a very common device among established developers, my guess would be it's well outside of the price range of students/contributors/etc).

**Comment by cdrini** [comment]
Hey @tfmorris , any updates on this?

**Comment by cdrini** [comment]
This pr has gone stale for ~2mo; closing. Feel free to rebase + update :+1:

**Comment by tfmorris** [comment]
I provided updates on Slack to confirm that the problem actually was real, but I really don't care. I'll chock it up to wasted volunteer work. Someone else will rediscover and refix it later.

**Pull request closed without merging**